checker/tests/nullness/DaikonEnhancedFor.java

```diff
@@ -0,0 +1,30 @@
+// Based on a false positive encountered in Daikon related to common CFGs
+// by the KeyFor checker.
+
+import java.util.*;
+import org.checkerframework.checker.nullness.qual.*;
+
+class DaikonEnhancedFor {
+
+    @SuppressWarnings("nullness")
+    Map<Object, Set<@KeyFor("cmap") Object>> cmap = null;
+
+    @SuppressWarnings("nullness")
+    Object[] getObjects() {
+        return null;
+    }
+
+    void process(@KeyFor("this.cmap") Object super_c) {
```

Comment:
create another copy of this test case with expected warnings for the lack of "this"